### ISSUE TITLE
[Bug]: Inconsistency in color usage on watch page

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#6472]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

There are multiple colors being used for `Show more`, `Show less`, `Click to view comments` and `View replies`. 

> @efb4f5ff-1298-471a-8973-3d47447115dc The page links are the `--link-color` variable which is just the `--accent-color` (Secondary Theme Color) - not sure why we have this one-time-use `--link-color` variable, but oh well. The comments button is `--title-color` which is base theme-dependent. For some reason, changing the secondary theme color is changing the rule for the comments button, which shouldn't be happening. You can inspect this more closely with the Dev Tools to see which CSS rules it's applying to figure out where that bug is coming from.

https://github.com/user-attachments/assets/befb4925-834a-41e0-82c4-a5edd9156bba

https://github.com/user-attachments/assets/965c6e62-6e41-4d6c-b7b5-a32ae627aa7f

### Expected Behavior

Dont use secondary color theme for  `Show more`, `Show less`, `Click to view comments` and `View replies`. 

### Issue Labels

inconsistent behavior, visual bug

### FreeTube Version

https://github.com/FreeTubeApp/FreeTube/actions/runs/12535178475

### Operating System Version

Win10 22H2

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by kommunarr** [comment]
It looks like this is due to both of the Catppucchin themes setting their `title-color` to `accent-color`. CC: @DontBlameMe99

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Dracula looks a bit purple-ish is that expected?

**Comment by kommunarr** [comment]
Yes, that theme's title color is [bd93f9](https://www.colorhexa.com/bd93f9)

**Comment by DontBlameMe99** [comment]
Is this something that should be reverted? If yes to which colors? I would be able to do this once #6468 is merged?

**Comment by kommunarr** [comment]
The gist is to use what best color a link/URL/`<a>` would be in that theme. Does that make sense?

**Comment by DontBlameMe99** [comment]
> The gist is to use what best color a link/URL/`<a>` would be in that theme. Look at how the Light, Dark, and Dracula themes handle that for three examples of the approaches you could take.

That makes sense. Should I change this in #6468?

**Comment by kommunarr** [comment]
Separate bug fix PR would be ideal

**Comment by DontBlameMe99** [comment]
After it gets merged ig? Then I'll make a PR which fixes both of them?

**Comment by kommunarr** [comment]
If that's what you prefer, sure! I tend to open multiple PRs and just resolve any merge conflicts based on when they're merged

**Comment by DontBlameMe99** [comment]
> If that's what you prefer, sure! I tend to open multiple PRs and just resolve any merge conflicts based on when they're merged

I could also do this I guess. I thought one PR doing this after it has been merged would be more elegant. But if you think otherwise I can do this as well.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
tbh you can do whatever you prefer. Small PR's changes like this one will probably get merged faster than a whole theme

**Comment by kommunarr** [comment]
Related issue that we should switch to the `link-color` for the About page links in the same effort due to it being the only apparent instance of mixing the theme colors with a `bg-color` background, thus failing color contrast issues. Such a fix should check that this succeeds in achieving a sufficient color contrast in this case, otherwise another solution will have to be pursued. [More information here.](https://github.com/FreeTubeApp/FreeTube/pull/6468#issuecomment-256554560)
